README.md

```diff
@@ -165,41 +138,41 @@ conda config --add channels conda-forge
 conda config --set channel_priority strict
 ```
 
-Once the `conda-forge` channel has been enabled, `_py-xgboost-mutex, _r-xgboost-mutex, libxgboost, py-xgboost, py-xgboost-cpu, py-xgboost-gpu, r-xgboost, r-xgboost-cpu, r-xgboost-gpu, xgboost` can be installed with `conda`:
+Once the `conda-forge` channel has been enabled, `libxgboost, py-xgboost, r-xgboost, xgboost` can be installed with `conda`:
```

Comment:
Do we still have the CPU and GPU variants?

Reply:
It is already possible for users to do this with the build string. For example

```
conda create -n xgboost 'libxgboost=*=cuda*' xgboost
```

This also what [conda-forge recommends with other CUDA packages like PyTorch and TensorFlow]( https://conda-forge.org/docs/user/tipsandtricks/#installing-cuda-enabled-packages-like-tensorflow-and-pytorch )

<hr>

Side note: The conda-forge package requires either an installed CUDA driver or setting [the environment variable `CONDA_OVERRIDE_CUDA`]( https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-virtual.html#overriding-detected-packages ). So on a system without the CUDA driver this would look like

```
export CONDA_OVERRIDE_CUDA=12.8
conda create -n xgboost 'libxgboost=*=cuda*' xgboost
```

Reply:
@jakirkham Could you please elaborate on the other way around, installing CPU-only package on a CUDA machine? We need to update the document in XGBoost: https://github.com/dmlc/xgboost/blob/master/doc/install.rst#conda .

Reply:
Certainly have edited my comment above to breakdown a few use cases with examples

Should add what I described above works even for the XGBoost 2 packages. Have tested this myself and would encourage you to try as well

Am also happy to do a PR upstream to update the docs

Reply:
Thank you for sharing, this is really useful!

Reply:
Ofc! 🙂

Tried to revise the docs with PR: https://github.com/dmlc/xgboost/pull/11404

Happy to iterate